refinery/tool_manager/test_data/visualization_LIST_igv.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Test LIST Visualization",
+  "tool_type": "VISUALIZATION",
+  "annotation": {
+    "container_name": "IGV_TEST",
+    "docker_image_name": "gehlenborglab/docker_igv_js:refinery_integration",
+    "description": "This visualization does really cool things... with IGV",
+    "parameters": [
+      {
+        "name": "Genome Build Param",
+        "description": "Genome Build Param description",
+        "value_type": "GENOME_BUILD",
+        "default_value": "hg19"
+      },
+      {
+        "name": "Attribute Param",
+        "description": "Attribute Param description",
+        "value_type": "ATTRIBUTE",
+        "default_value": "Species"
+      },
+      {
+        "name": "File Param",
+        "description": "File Param description",
+        "value_type": "FILE",
+        "default_value": "/media/file_store/file.cool"
+      }
+    ],
+    "output_files":[],
+    "file_relationship": {
+      "file_relationship": {},
+      "value_type": "LIST",
+      "name": "Flat list of N Samples",
+      "input_files": [
+        {
+          "allowed_filetypes": [{"name": "SEG"}],
+          "name": "Cool Input File",
+          "description": "Cool Input File Description"
```

Comment:
This is just the ToolAnnotation, UUIDs are assigned when we create a ToolDefinition and put things into the Database